extensions/sql-migration/src/dialog/migrationCutover/migrationCutoverDialog.ts

```diff
@@ -435,6 +435,19 @@ export class MigrationCutoverDialog {
 		}
 	}
 
+	private getMigrationDetails(): string {
+		if (this._model.migrationOpStatus) {
```

Comment:
Can you also use this method in the place where you got the code from?

https://github.com/microsoft/azuredatastudio/blob/e75572f5ca7e0b221141841e208679ee701ab455/extensions/sql-migration/src/dialog/migrationCutover/migrationCutoverDialog.ts#L308-L308

Reply:
Done, Thanks!